scripts/download-azure/main.go

```diff
@@ -56,7 +56,12 @@ func getWorkingDirectory() (string, error) {
 }
 
 func getDownloadURL(ctx context.Context) (string, error) {
```

Comment:
maybe worth renaming this dir to something like `download-azure-ip-range`, as the repo grew in size and contains many disconnected things